js/node/CMakeLists.txt

```diff
@@ -86,61 +96,29 @@ else()
 endif()
 
 if (WIN32)
-  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime.lib)
+  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime)
 elseif (APPLE)
   target_link_libraries(onnxruntime_binding PRIVATE libonnxruntime.${ort_version}.dylib)
   set_target_properties(onnxruntime_binding PROPERTIES INSTALL_RPATH "@loader_path")
 else()
-  target_link_libraries(onnxruntime_binding PRIVATE libonnxruntime.so.${ort_version})
+  target_link_libraries(onnxruntime_binding PRIVATE onnxruntime)
   set_target_properties(onnxruntime_binding PROPERTIES INSTALL_RPATH "$ORIGIN/")
 endif()
 
-# post build
-
-add_custom_command(
-  TARGET onnxruntime_binding POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E make_directory ${dist_folder}
-    COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:onnxruntime_binding> ${dist_folder}
-)
 
 if (WIN32)
-  add_custom_command(
-    TARGET onnxruntime_binding POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy
-      ${ONNXRUNTIME_WIN_BIN_DIR}/onnxruntime.dll
-      ${dist_folder}
-  )
+  file(COPY ${ONNXRUNTIME_WIN_BIN_DIR}/onnxruntime.dll
+      DESTINATION ${dist_folder})
   if (USE_DML)
-    add_custom_command(
-      TARGET onnxruntime_binding POST_BUILD
-      COMMAND ${CMAKE_COMMAND} -E copy
-      ${ONNXRUNTIME_WIN_BIN_DIR}/DirectML.dll
-      ${dist_folder}
-    )
+    file(COPY ${ONNXRUNTIME_WIN_BIN_DIR}/DirectML.dll
+      DESTINATION ${dist_folder})
   endif ()
-  if (CMAKE_BUILD_TYPE STREQUAL "Debug")
-    add_custom_command(
-      TARGET onnxruntime_binding POST_BUILD
-      COMMAND ${CMAKE_COMMAND} -E copy
-        ${ONNXRUNTIME_WIN_BIN_DIR}/onnxruntime.pdb
-        ${dist_folder}
-      COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE_DIR:onnxruntime_binding>/onnxruntime_binding.pdb ${dist_folder}
-    )
-  endif()
 elseif (APPLE)
-  add_custom_command(
-    TARGET onnxruntime_binding POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy
-      ${ONNXRUNTIME_BUILD_DIR}/libonnxruntime.${ort_version}.dylib
-      ${dist_folder}
-  )
+  file(COPY ${ONNXRUNTIME_BUILD_DIR}/libonnxruntime.dylib
+      DESTINATION ${dist_folder} FOLLOW_SYMLINK_CHAIN)
```

Comment:
The "FOLLOW_SYMLINK_CHAIN" keyword means: if the file we are copying is a symlink point to somewhere, we will copy the symlink and the actually file to the destination and keep the symbolic link relationship.